charts/rancher-backup/files/default-resourceset-contents/rancher.yaml

```diff
@@ -36,6 +36,15 @@
   resourceNameRegexp: "management.cattle.io$|project.cattle.io$|catalog.cattle.io$|resources.cattle.io$"
 - apiVersion: "management.cattle.io/v3"
   kindsRegexp: "."
+  excludeKinds:
```

Comment:
For some reason go regxp doesn't support exclude, and also it might be quite expensive to support that. So I am just adding this.

Reply:
Yup, Go's flavor of regexp doesn't support negative lookahead which makes is (nearly?) impossible to build regexps that match "anything except <word1>, <word2>, ...": https://stackoverflow.com/a/26792316/16564280
Just wanted ti share the link for more context.